tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -22,21 +22,18 @@
 from __future__ import division
 from __future__ import print_function
 
-import collections
```

Comment:
Appreciate the import cleanup, but it’d be best to separate that out,
particularly because it should be accompanied by removing the build
edges (`//tensorboard/util:tensor_util`, et al.). Can you revert this
part of the diff? I can take care of the cleanup if you like.

Reply:
Removed.